master/static/migrations/20241010174718_alter-user-session-add-access-token-info.tx.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TYPE token_type AS ENUM ('USER_SESSION', 'ACCESS_TOKEN');
+
+ALTER TABLE user_sessions ADD COLUMN created_at TIMESTAMP DEFAULT NULL;
+ALTER TABLE user_sessions ADD COLUMN token_type token_type NOT NULL DEFAULT 'USER_SESSION';
+ALTER TABLE user_sessions ADD COLUMN revoked boolean DEFAULT false NOT NULL;
```

Comment:
I'd prefer the use of a nullable timestamp rather than a `boolean`—both in this case and in general. Like [this author](https://changelog.com/posts/you-might-as-well-timestamp-it), I've never regretted it, and it's generally considered good practice. You can always write queries like `SELECT revoked IS NULL FROM user_sessions` if you want it in boolean form in the API layer, or check for nullity in the CLI/WebUI code if you want a (nullable) timestamp in the API but presented to users as a boolean.